docs/release-notes/Percona-Everest-1.5.0-(2025-02-07).md

```diff
@@ -0,0 +1,154 @@
+
+- [EVEREST-1795](https://perconadev.atlassian.net/browse/EVEREST-1795): When attempting to update a namespace using `everestctl` that was created with `kubectl` (not managed by Percona Everest), the error message was unclear. It did not provide actionable steps for the user to resolve the issue. We have improved the error message to give more insights into the issue.
+
+
+## Bugs
+
+- [EVEREST-1261](https://perconadev.atlassian.net/browse/EVEREST-1261): We fixed an issue where a user who had already added a backup storage location received an incorrect error message when trying to add another one with the same bucket name and URL.
+
+
+- [EVEREST-1401](https://perconadev.atlassian.net/browse/EVEREST-1401):
+Now, when you create/edit the database cluster with sharding enabled for PSMDB, it will display the correct resources required for the specified number of shards.
+
+- [EVEREST-1537](https://perconadev.atlassian.net/browse/EVEREST-1537):
+We have resolved an issue that caused Percona Everest uninstallation to fail when attempting to delete database clusters due to a timeout.
+
+
+- [EVEREST-1581](https://perconadev.atlassian.net/browse/EVEREST-1581): The database remained in a **Deleting** state, despite all components being deleted. The issue has now been resolved.
+
+- [EVEREST-1588](https://perconadev.atlassian.net/browse/EVEREST-1588): We have fixed an issue where the PostgreSQL database was stuck in an **initializing** state after a restore.
+
+- [EVEREST-1589](https://perconadev.atlassian.net/browse/EVEREST-1589): We have fixed an issue where the MySQL database reamined stuck in an **initializing** state for 1-node cluster.
+
+- [EVEREST-1590](https://perconadev.atlassian.net/browse/EVEREST-1590) Installation is stuck
+
+- [EVEREST-1647](https://perconadev.atlassian.net/browse/EVEREST-1647): When creating a monthly schedule on day 1 at 12:00 AM (the default option when choosing **Monthly**), it led to an error for PSMDB. The issue has been resolved now.
+
+- [EVEREST-1674](https://perconadev.atlassian.net/browse/EVEREST-1674): The message **Enforce did not pass** appeared randomly in the UI. Additionally, databases in the UI sometimes disappeared and then reappeared after a few seconds. We have now resolved the issue.
+
+
+- [EVEREST-1724](https://perconadev.atlassian.net/browse/EVEREST-1724): Sharding no longer resets to its default setting (disabled) when navigating back in the database creation wizard.
+
+
+- [EVEREST-1728](https://perconadev.atlassian.net/browse/EVEREST-1728): The database dashboard now updates automatically when you create databases in different namespaces across various browser sessions.
+
+
+- [EVEREST-1729](https://perconadev.atlassian.net/browse/EVEREST-1729): Installation fails if 1.3.0 release cli is used to install 1.4.0
```

Comment:
We added a mechanism to prevent a given CLI version to install incompatible everest versions.